libs/utils/env.py

```diff
@@ -439,6 +439,51 @@ def _init_target(self, force = False):
             self.__modules = ['bl', 'cpufreq']
             platform = Platform(model='pixel')
 
+        # Initialize gem5 platform
+        elif self.conf['board'].upper() == 'GEM5':
```

Comment:
Since this is also a pretty long setup compare to the other cases above, perhaps we could better move this code into a dedicated function. something like:
```python
elif self.conf['board'].upper() == 'GEM5':
    self.nrg_model = None
    self.__modules=['cpufreq']
    platform = self._init_target_gem5()

#... further down in this file

def _init_target_gem5(self):
    # This branch code
   return platform
```